### ISSUE TITLE
[bug] make help not working in ubuntu linux system

### ISSUE THREAD

**Issue by jim-wang-intel** [edgexfoundry/developer-scripts#415]
`make help` does not show the intended-to-be service list, but it produces the error message instead:

```sh
jim@jim-NUC7i5DNHE:~/go/src/github.com/edgexfoundry/developer-scripts/compose-builder$ make help
- arm64 no-secty dev app-dev -
make: mqtt-bus: Command not found
Makefile:215: recipe for target 'help' failed
make: [help] Error 127 (ignored)
make: taf-secty: Command not found
Makefile:215: recipe for target 'help' failed
make: [help] Error 127 (ignored)
make: ds-bacnet: Command not found
Makefile:215: recipe for target 'help' failed
make: [help] Error 127 (ignored)
make: asc-http: Command not found
Makefile:215: recipe for target 'help' failed
make: [help] Error 127 (ignored)
make: modbus-sim: Command not found
Makefile:215: recipe for target 'help' failed
make: [help] Error 127 (ignored)
See README.md in this folder

```

**Comment by lenny-intel** [comment]
Occurs on Windows also. I left a debug `echo ${OPTIONS}` which needs to be removed.

**Comment by jim-wang-intel** [comment]
> Occurs on Windows also. I left in the debug `echo ${OPTIONS}` which needs to be removed.

ok, i think it can be fixed if using "$$OPTIONS" instead of just ${OPTIONS}
I am fine with removing it altogether.

**Comment by lenny-intel** [comment]
> ok, i think it can be fixed if using "$$OPTIONS" instead of just ${OPTIONS}
> I am fine with removing it altogether.

I isn't useful. I just had it there for debugging and forgot to remove it.

**Comment by jim-wang-intel** [comment]
will be Closed by PR #414
